docs/docs/api/hooks/useFrameCallback.md

```diff
@@ -36,9 +37,19 @@ Properties:
                     or not (`false`)
 * `callbackId: number`: a unique identifier of the callback function
 
+#### `FrameInfo: [object]`
+
+Properties:
+* `timestamp: number`: the system time (in milliseconds) when the last
+  frame was rendered
+* `duration: number | undefined`: time (in milliseconds) since last frame - this value
+  will be `undefined` on the first frame after activation. Starting from the second frame,
```

Comment:
I think we should use `null` here instead of `undefined`. What do you think?
```suggestion
* `timeSinceLastFrame: number | null`: time (in milliseconds) since last frame - this value
  will be `null` on the first frame after activation. Starting from the second frame,
```